projects/precise/src/environments/environment.ts

```diff
@@ -28,7 +28,7 @@ export const environment = {
         forum: 'https://community.mycroft.ai',
         marketplace: 'http://market.mycroft.test',
         mimic: 'http://mimic.mycroft.ai',
```

Comment:
There seems to be a handful still using http and some using the `mycroft.test` domain

Reply:
This has been resolved.

Reply:
Fixed